### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by LilianBoulard**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,8 +36,8 @@ for which we usually identify three common cases: **similarities, typos and vari
 `The first example notebook <https://skrub-data.github.io/stable/auto_examples/01_dirtycategories.html>`_
 goes in-depth on how to identify and deal with dirty data using the `skrub` library.
 
-What `skrub` does not
-~~~~~~~~~~~~~~~~~~~~~~~~~
+What `skrub` cannot do
```
Comment (Vincent-Maladiere) [inline README.rst:39]: Maybe nitpick: I'm in favor of keeping "does not". It feels like the natural way of saying "We can't use skrub to do this". WDYT?

File: README.rst [hunk 2]
```diff
@@ -51,9 +53,10 @@ methods.
 Installation (WIP)
 ------------------
 
-Once the first release is done, skrub can be easily installed via `pip`::
+There are currently no PiPy releases.
+You can still install the package from the GitHub repository with:
 
-    pip install skrub
+    pip install git+https://github.com/skrub-data/skrub.git
```
Comment (Vincent-Maladiere) [inline README.rst:59]: After an IRL discussion with @GaelVaroquaux, our strategy is to be tested and used only by people with a significant technical and statistical grasp during the early days of skrub. These people will know how to install and test the project without us helping them. We can discuss that further.

File: README.rst [hunk 3]
```diff
@@ -1,12 +1,13 @@
 `skrub`
-===========
+=======
 
 .. image:: https://skrub-data.github.io/stable/_static/skrub.svg
    :align: center
+   :width: 50 %
```
Comment (Vincent-Maladiere) [inline README.rst:6]: 👍

**Review by Vincent-Maladiere — COMMENTED** [review]
Some quick reviews :)

File: README.rst (hunk 2 above)
Comment (jovan-stojanovic) [reply README.rst:59]: I guess we revert this when skrub will be available in pypi

**Review by Vincent-Maladiere — APPROVED** [review]
This LGTM :)

**Comment by LilianBoulard** [comment]
The shields' centering doesn't work for now, I'll turn this PR to draft until I fix this

**Comment by Vincent-Maladiere** [comment]
I'm more in favor of merging this PR now and having another PR later to center the badges. IMHO, it's not as important as correcting the logo size or the link.

**Comment by LilianBoulard** [comment]
Sounds reasonable, merging!

**Pull request merged**